src/main/java/nl/tudelft/contextproject/gui/CameramanLiveController.java

```diff
@@ -1,373 +1,205 @@
-        smallView.setImage(current);
     }
 
     /**
-     * Initialize button styling and functionality.
+     * Initialize button functionality.
      */
-    private void initializeButtons() {
-        initSwapButton();
-
+    private void initButtons() {
         btnBack.setOnAction(event -> {
             MenuController.show();
         });
-
-        btnConfirm.setOnAction(event -> {
-            if (!speed.getText().isEmpty()) {
-                int spd = Integer.parseInt(speed.getText());
-
-                if (spd > 100) {
-                    spd = 100;
-                }
-
-                if (live) {                   
-                    liveSpeed = spd;
-                    speed.setText(String.valueOf(spd));
-                } else {
-                    currentSpeed = spd;
-                    speed.setText(String.valueOf(spd));
-                }
-            }
-        });
-    }
-
-    /**
-     * Initializes the swap button.
-     */
-    private void initSwapButton() {
-        swap.setOnAction(event -> {
-            Image three = bigView.getImage();
-            bigView.setImage(smallView.getImage());
-            smallView.setImage(three);
-            String text = bigStatusLabel.getText();
-            bigStatusLabel.setText(smallStatusLabel.getText());
-            smallStatusLabel.setText(text);
-
-            if (text.equals("LIVE")) {
-                smallStatusLabel.setStyle("-fx-text-fill: red;");
-                bigStatusLabel.setStyle("");
-            } else {
-                smallStatusLabel.setStyle("");
-                bigStatusLabel.setStyle("-fx-text-fill: red;");
-            }
-
-            swapCurrentView();
-            swapInfoTable();
-            swapSpeed();
-        });
-    }
-
-    /**
-     * Initialises movements.
-     */
-    private void initializeMovements() {
-        moveOne.setOnAction(event -> {
-            movement.setText(moveOne.getText());
-            setMovement(MovementType.ZOOM_IN);
-        });
-
-        moveTwo.setOnAction(event -> {
-            movement.setText(moveTwo.getText());
-            setMovement(MovementType.ZOOM_OUT);
-        });
-
-        moveThree.setOnAction(event -> {
-            movement.setText(moveThree.getText());
-            setMovement(MovementType.LEFT);
-        });
-
-        moveFour.setOnAction(event -> {
-            movement.setText(moveFour.getText());
-            setMovement(MovementType.RIGHT);
-        });
-
-        moveFive.setOnAction(event -> {
-            movement.setText(moveFive.getText());
-            setMovement(MovementType.UP);
-        });
-
-        moveSix.setOnAction(event -> {
-            movement.setText(moveSix.getText());
-            setMovement(MovementType.DOWN);
-        });
-
-        moveSeven.setOnAction(event -> {
-            movement.setText(moveSeven.getText());
-            setMovement(MovementType.CUSTOM);
-        });
         
+        //TODO remove temp button
```

Comment:
@tkolenbrander It could be useful to keep it for now, to enable easy demo-ing of the current shot highlight. It will be done with the database connection once that is finished and merged, and then then button can be removed